test/byContract/IexecEscrow/IexecEscrowNative.ts

```diff
@@ -0,0 +1,302 @@
+// SPDX-FileCopyrightText: 2020-2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
+// SPDX-License-Identifier: Apache-2.0
+
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { BigNumber } from 'ethers';
+import { ethers, expect } from 'hardhat';
+import CONFIG from '../../../config/config.json';
+import { loadHardhatFixtureDeployment } from '../../../scripts/hardhat-fixture-deployer';
+import { IexecInterfaceNative, IexecInterfaceNative__factory } from '../../../typechain';
+import { getIexecAccounts } from '../../../utils/poco-tools';
+
+const DEPOSIT_AMOUNT = BigNumber.from(100);
+const DEPOSIT_AMOUNT_ETHER = ethers.utils.parseUnits(DEPOSIT_AMOUNT.toString(), 9);
```

Comment:
We can talk about that, I think `const` "uppercases" are more for global/app variables (`DAYS_OF_WEEK=7`, `PI=3.14`, `PUBLIC_KEY=..`, `FINAL_POCO_PERIOD=10`), I would eventually see something like:
```
const depositAmount = ..
const etherDepositAmount = ..
const ZERO_ADDRESS = ..
```

Reply:
Done thanks